docs/en/advanced_guides/index.rst

```diff
@@ -25,3 +25,4 @@ Customization
    customize_dataset.md
    customize_models.md
    customize_runtime.md
+   pure_point_cloud_dataset.md
```

Comment:
We will remove `pure_point_cloud_dataset.md` in dev-1.x and it will be merged into `customize_dataset.md`